README.md

```diff
@@ -88,6 +87,26 @@ Then in HTML
 
 Note: Input can be a HTML string or a jsonDoc
 
+#### Reactive Form Usage
```

Comment:
Let's create this in the docs as a section/page, working with Reactive Forms. We can add this there.

Thanks